src/main/steps/common/alldocuments/template.njk

```diff
@@ -10,7 +10,7 @@
   </h1>
   <div class="govuk-form-group">
     <fieldset class="govuk-fieldset">
-      <p class="govuk-body">{{caseNumber}} #{{userCase.id}}</p>
+      <p class="govuk-body">{{caseNumber}}  &nbsp#{{userCase.id}}</p>
```

Comment:
Remove hash and have space between caseNumber text and case id throughout the application

Reply:
Done this.